docker/Dockerfile

```diff
@@ -0,0 +1,83 @@
+ARG BASE_IMAGE=ubuntu:20.04
+FROM ${BASE_IMAGE}
+
+# Necessary to install tzdata. It will default to UTC.
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y \
+      python3.9 \
+      # python3.9-distutils is required to install pip
+      # it unfortunately has to install python3.8-minimal as well...
+      python3.9-distutils \
+      # python-is-python3 creates a symlink for python to python3
+      python-is-python3 \
+      # For creating virtual environments
+      python3.9-venv \
+      # wget is used to obtain pip
+      wget \
+      # apache
+      apache2 \
+      apache2-dev \
+      libapr1-dev \
+      apache2-utils \
+      gosu && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set python3 to python3.9 (otherwise, it will be python3.8)
+RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 1
+
+# Never use a cache directory for pip, both here in this Dockerfile
+# and when we run the container.
+ENV PIP_NO_CACHE_DIR=1
+
+# Install and upgrade pip
+RUN wget -O- https://bootstrap.pypa.io/get-pip.py | python3.9 && \
+    pip install -U pip
+
+# Install setup dependencies
+RUN pip install PyYAML
+
+# Set up needed permissions and users
+RUN groupadd trame-user -g 1000 && \
+    groupadd proxy-mapping -g 1001 && \
+    useradd -u 1000 -g trame-user -G proxy-mapping -s /sbin/nologin trame-user && \
+    usermod -a -G proxy-mapping www-data && \
+    mkdir -p /opt/trame/log && \
```

Comment:
Do you know if this log file is being used for anything? Or can we delete it?

Reply:
no as we use the /deploy/server/logs instead

Reply:
Fixed.